test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SqlProjects/SqlProjectsServiceTests.cs

```diff
@@ -304,7 +304,7 @@ await service.HandleAddSystemDatabaseReferenceRequest(new AddSystemDatabaseRefer
             {
                 ProjectUri = projectUri,
                 SystemDatabase = SystemDatabase.MSDB,
-                DatabaseVariable = "$(EmEssDeeBee)",
+                DatabaseLiteral = "$(EmEssDeeBee)",
```

Comment:
I thought DatabaseLiterals don't need the `$()`

Reply:
Oh yeah, good catch.